### PR TITLE
Provide a tip for installing asciidoc on Windows

### PR DESCRIPTION
**Pull request by ltearno**
About the code page change that is required when working in windows command line

### CODE REVIEW

File: index.adoc [hunk 1]
```diff
@@ -116,6 +116,8 @@ TIP: The benefit of using a Linux package manager to install the gem is that it
 The drawback is that the package may not be available immediately after the release of the gem.
 If you need the latest version, you can always fallback to using the `gem` command.
 
+TIP: If you are working on Windows, you may get this error : `ERROR:  While executing gem ... (Encoding::UndefinedConversionError)`. In that case, you have to change the code page used by your command line. This is done by executing `chcp 1252`. Then use the following commands as stated.
```
Comment (mojavelinux) [inline index.adoc]: I like this suggestion, but Asciidoctor strongly recommends the use of UTF-8, so this should be:

```
$ chcp 65001
```

**Comment by ltearno** [comment]
Tested and approved !
I updated the pull request to instruct the user to use the 65001 code page.

As I am curious, would you please kindly point to some resource explaining why 65001 and not 1252 ?

Thanks

**Comment by mojavelinux** [comment]
65001 is the code for UTF-8, whereas 1252 is the code for Microsoft's extended ASCII (a blend of US-8859-1 + commonly used special symbols). UTF-8 is a far better choice. 1252 only gives you a few extra miles of gas. See https://en.wikipedia.org/wiki/Windows-1252

**Comment by mojavelinux** [comment]
Thanks for adding this info!

**Pull request merged**